spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/feign/FeignClientsConfiguration.java

```diff
@@ -56,6 +58,9 @@
 	@Autowired(required = false)
 	private List<AnnotatedParameterProcessor> parameterProcessors = new ArrayList<>();
 
+	@Autowired(required = false)
+	private List<FeignCustomizer> feignCustomizers = new ArrayList<>();
```

Comment:
`FeignCustomizer` seems generic like it could be used to customize the builder, maybe `FeignFormatterRegistrar`?